### ISSUE TITLE
PDF Viewer does not render Chinese Characters

### ISSUE THREAD

**Issue by SparklyYS** [iamhyc/Overleaf-Workshop#34]
I would like to express my sincere gratitude for the hard work and dedication put into this plugin. However, I've encountered an issue with the current version, which is the inability to correctly display Chinese characters in the generated PDF. This poses a challenge for users like me who work primarily on projects written in Chinese. Therefore, I kindly request the addition of support for displaying Chinese characters in the PDF output.

**Comment by iamhyc** [comment]
Too wordy. Just show us: 1) what is wrong, and 2) what is the expected result.

I guess you mean your project with Chinese characters cannot compile. If that is the case, please switch to `xelatex` on the webpage firstly and try again.

**Comment by iamhyc** [comment]
I think the expected output pdf is like the following example. The problem is fixed in c03146b and the fix will be included in next release, `v0.4.1`.

![Screenshot](https://github.com/iamhyc/Overleaf-Workshop/assets/9068301/419dbb6c-4a64-493b-9b41-0e084aa6ef58)

**Comment by iamhyc** [comment]
Since v0.4.1, you can also switch the spell check in vscode. I see the issue as resolved.

![Screenshot from 2023-10-30 20-36-08](https://github.com/iamhyc/Overleaf-Workshop/assets/9068301/5ad3a098-ce3b-48f0-b79d-baf92177500f)

**Comment by iamhyc** [comment]
The version v0.4.1 is released. I see this issue as resolved.
